src/components/PublishContainer/PublishedScreen/PublishedScreen.tsx

```diff
@@ -51,42 +57,80 @@ export const PublishedScreen: FunctionComponent<PublishScreen> = ({
     };
   });
 
+  const getDisplayTitle = (): ReactElement => {
+    switch (publishState) {
+      case PUBLISH_STATE.PENDING_CONFIRMATION:
+        return (
+          <>
+            <div className="h-6 w-6 mr-2">
+              <PublishLoader />
+            </div>
+            Publishing document(s)...
+          </>
+        );
+
+      case PUBLISH_STATE.CONFIRMED:
+        if (publishedDocuments.length > 0) {
+          return (
+            <>
+              <CheckCircle className="mr-2 text-teal" />
+              Document(s) issued successfully
+            </>
+          );
+        } else {
+          return (
+            <>
+              <XCircle className="mr-2 text-red" />
+              Document(s) failed to issue
+            </>
+          );
+        }
+
+      default:
+        return <>Please wait while we prepare your document(s)</>;
+    }
+  };
+
   return (
     <Container>
       <div className="container mx-auto pt-8">
         <ProgressBar step={3} />
         <div className="flex justify-between items-end">
-          <Title className="flex items-center mb-8">
-            <CheckCircle className="mr-2 text-teal" />
-            {publishedDocuments.length > 0
-              ? "Document(s) issued successfully"
-              : "Document(s) failed to issue"}
-          </Title>
-          <div>
-            <Button className="bg-white text-orange px-4 py-3 mb-6 mr-4" onClick={createAnotherDoc}>
-              Create another Document
-            </Button>
-            <Button
-              className="bg-orange text-white self-end py-3 px-4 mb-6"
-              data-testid="form-logout-button"
-              onClick={onDone}
-            >
-              Logout
-            </Button>
-          </div>
+          <Title className="flex items-center mb-8">{getDisplayTitle()}</Title>
+          {publishState === PUBLISH_STATE.CONFIRMED && (
+            <div>
+              <Button
+                className="bg-white text-orange px-4 py-3 mb-6 mr-4"
+                onClick={createAnotherDoc}
+              >
+                Create another Document
+              </Button>
+              <Button
+                className="bg-orange text-white self-end py-3 px-4 mb-6"
+                data-testid="form-logout-button"
+                onClick={onDone}
+              >
+                Logout
+              </Button>
+            </div>
+          )}
         </div>
       </div>
       <div className="bg-lightgrey-lighter p-6 h-screen">
-        {publishedDocuments && publishedDocuments.length > 0 && (
+        {(publishState === PUBLISH_STATE.PENDING_CONFIRMATION ||
+          (publishedDocuments && publishedDocuments.length > 0)) && (
```

Comment:
```
(publishedDocuments?.length > 0 || pendingPublishDocuments?.length > 0)
```

Reply:
In fact I dont think need the `?` since the type is showing it cannot be undefined, will at least be an empty array.

Reply:
ok